### DISCUSSION TITLE
[Feature] Icon on image if description

### DISCUSSION THREAD

**Discussion by TobiZog** [immich-app/immich#7778] [Feature Request]
### The feature

In album view are icons for videos and 360° images displayed in the right top corner of an image. It would be nice, if there were also an icon, if the image has a description.

Here is an example:

![example](https://github.com/immich-app/immich/assets/67568441/1b8a0ccf-02a0-410b-9c35-7b1dfde97594)

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#21029

**Reply by jeanius7** [reply]
This would be awesome. Google does this by visually showing you have added a description to the image. I use the description a LOT to help with searches. Sometimes searching is okay, but putting descriptions in to search is 100% better.

Would love to see this on Server/Web/Mobile. 

For instance:
<img width="435" height="307" alt="image" src="https://github.com/user-attachments/assets/c48fce22-6f15-44aa-9e90-0ab2045cdc01" />
